src/managers/GuildMemberManager.js

```diff
@@ -249,6 +262,12 @@ class GuildMemberManager extends CachedManager {
       _data.channel = undefined;
     }
     _data.roles &&= _data.roles.map(role => (role instanceof Role ? role.id : role));
+
+    if (typeof _data.communicationDisabledUntil !== 'undefined') {
+      _data.communication_disabled_until =
+        _data.communicationDisabledUntil && new Date(_data.communicationDisabledUntil).toISOString();
```

Comment:
Does this set it to null if we provide null? Asking to be sure

Reply:
Yep! `null && true` for example will return `null`. In this case, providing `null && new Date(null)` will return `null`.